renovate.json5

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": [
+    // update base images in Dockerfiles
+    ":docker",
+    ":disableDependencyDashboard",
```

Comment:
Why disable it? It's a good way to check if Renovate is discovering the places to update correctly.

Reply:
Done, put it back